kuma/redirects/models.py

```diff
@@ -0,0 +1 @@
+from django.db import models
```

Comment:
I believe the file is required for django to consider it an "app" (that could be old info though). It can be empty though. Looks like the default models.py when you use `manage.py startapp`.